contracts/MultiSend.sol

```diff
@@ -1,74 +1,117 @@
+// Copyright (C) 2018 Alon Bukai This program is free software: you 
+// can redistribute it and/or modify it under the terms of the GNU General 
+// Public License as published by the Free Software Foundation, version. 
+// This program is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+// or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details. You should have received a copy of the GNU General Public
+// License along with this program. If not, see http://www.gnu.org/licenses/
```

Comment:
I loves me some GPL.

Reply:
I `stole` this from you 😄

Reply:
It's not stealing if it's GPL licensed 😉